beacon_node/network/src/sync/range_sync/chain.rs

```diff
@@ -885,6 +885,8 @@ impl<T: BeaconChainTypes> SyncingChain<T> {
                     failing_batch: batch_id,
                 });
             }
+            // FIXME: hack to remove range request before a retry
+            network.remove_range_request_by_id(request_id);
```

Comment:
Note that this hack didn't fix it

<img width="936" height="273" alt="image" src="https://github.com/user-attachments/assets/f8f86078-380d-457e-9461-8440571e1d73" />

The metric i was looking at is `sync_active_network_requests`, type == `components_by_range`, recorded here:
https://github.com/sigp/lighthouse/blob/790c94c75a940dbdbf8dd5d354519c160719d780/beacon_node/network/src/sync/network_context.rs#L1705-L1708

This change what caused entries not to be removed i believe, its used by both range and backfill:

https://github.com/sigp/lighthouse/blob/790c94c75a940dbdbf8dd5d354519c160719d780/beacon_node/network/src/sync/network_context.rs#L745-L751